autoload/slime.vim

```diff
@@ -25,7 +25,8 @@ function! s:ScreenSend(config, text)
         \ " -X eval \"readreg p " . g:slime_paste_file . "\"")
   call system("screen -S " . shellescape(a:config["sessionname"]) . " -p " . shellescape(a:config["windowname"]) .
         \ " -X paste p")
-  call system('screen -X colon "
"')
+  call system('screen -X colon "
```

Comment:
any reason this is changed?

Reply:
oh probably i accidentally added a space. i closed the PR anyway so dont worry about it.